dbt/adapters/redshift/impl.py

```diff
@@ -65,6 +68,10 @@ class RedshiftAdapter(SQLAdapter):
         }
     )
 
+    @property
+    def _behavior_extra(self) -> List[BehaviorFlag]:
+        return [{"name": "retire_pg_catalog", "default": False}]
```

Comment:
Make sure this name aligns with the naming convention: https://github.com/dbt-labs/dbt-core/blob/main/docs/guides/behavior-change-flags.md#rules-for-introducing-a-new-flag